standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -454,6 +454,8 @@ public static ConfVars getMetaConf(String name) {
         "hive.metastore.acidmetrics.check.interval", 300,
         TimeUnit.SECONDS,
         "Time in seconds between acid related metric collection runs."),
+    METASTORE_ACIDMETRICS_EXT_ON("metastore.acidmetrics.ext.on", "hive.metastore.acidmetrics.ext.on", true,
+        "Whether to collect additional acid related metrics outside of the acid metrics service."),
```

Comment:
I think these are only enabled if `MetastoreConf.getBoolVar(conf, MetastoreConf.ConfVars.METRICS_ENABLED)==true` , so it would be good to mention that in the description

Reply:
And/Or HIVE_SERVER2_METRICS_ENABLED. Shall I add both?

Reply:
Yes, right:)

Reply:
fixed